AddOns/BasicFinancials/app/src/codeunit/InstallExtension.Codeunit.al

```diff
@@ -0,0 +1,37 @@
+codeunit 20603 "Install Extension BF"
+{
+    /*
+    Installation process 
+
+    1)	The partner creates a new Business Central Tenant
```

Comment:
To install the extension, the partner does the following:

1. Creates a new Business Central tenant.
2. Adds at least one user who has a Basic FInancials license assigned to them in the Azure Active Directory tenant.
3. Removes all companies manually, including Cronus.
4. Creates one production company that does not contain data or setups.
5. Imports a configuration package that includes Basic Financials setup data.
6. Installs the Basic Financials extension. During installation the extension verifies the country availability that there is only one company.
7. Completes the Basic Financials assisted setup guide. The assisted setup guide checks for the Basic Financials license.
8. Sends sign in information to the customer in an email.